mavros_msgs/msg/HilStateQuaternion.msg

```diff
@@ -0,0 +1,12 @@
+# HilStateQuaternion.msg
+#
+# ROS representation of MAVLink HIL_STATE_QUATERNION
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_STATE_QUATERNION
+
+std_msgs/Header header
+sensor_msgs/Imu imu
+geometry_msgs/Vector3 linear_velocity
+geographic_msgs/GeoPoint geo
```

Comment:
Why are you always asking for refactoring this? Me and @mzahana already tested this with V-REP sim and it is working